app/models/instance.rb

```diff
@@ -9,7 +9,7 @@ class << self
     #
     # @return [Instance]
     def default
-      @default ||= find_by(host: Instance::DEFAULT_HOST_NAME)
+      @default ||= find_by(id: DEFAULT_INSTANCE_ID)
```

Comment:
Here should be find_by because find will raise an exception when record does not exist.